backend/src/cache/cache.ts

```diff
@@ -0,0 +1,21 @@
+import { LRUCache } from "lru-cache";
+import { Analysis } from "@types";
+import { logMessage } from "../services/logging";
+
+const cache = new LRUCache<string, Analysis>({
+  max: 100,
+  ttl: 1000 * 60 * 5, // 5 minutes
```

Comment:
Out of curiosity, why did you land on "max: 100" and "ttl: 5mins"? I could see an argument for caching longer since playlist won't often change.

Reply:
This was just the autocomplete defaults. I've reset the cache to expire at 24 hours instead